src/api/hooks/useActiveSymbols.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react';
+import useQuery from './useQuery';
+import useLogin from './useLogin';
+
+const useActiveSymbols = () => {
+    const { is_logged_in, isAuthorized } = useLogin();
```

Comment:
I know it's out of the scope of this card but could u pls update this like this 🙏 
```suggestion
    const { isLoggedIn, isAuthorized } = useLogin();
```